metabrainz/templates/index/team.html

```diff
@@ -0,0 +1,134 @@
+    </div>
+  </div>
+  
+  <div class="people">
+    <div class="pic long">
+      <img src="{{  url_for('static', filename='img/zas.jpg') }}"/>
+    </div>
+    <div>
+      <h4><strong>System Administrator </strong>- Laurent Monin | zas</h4>
+      <p> A zealous music devotee, Laurent Monin first stumbled upon MusicBrainz and Picard in attempts to codify his vast collection of over 1500 CDs. In addition to being a music fanatic, Laurent has been programming on anything with more than 1 bit since 1982 and has been an esteemed Linux system administrator since 1998.
+      <br/><br/> With his extensive repertoire of skills and passions, Laurent has actively contributed to a myriad of open source projects, notably <a href="https://en.wikipedia.org/wiki/ELinks">ELinks</a> and <a href="https://en.wikipedia.org/wiki/Geeqie">Geeqie</a>, and loves to tinker with the hardware of various audio gadgets, including amplifiers and loudspeakers.
+       However, Laurent's contributions do not end there. He currently sustains the MetaBrainz servers while residing near Besançon, France. 
+      </p>
+    </div>
+  </div>
+
+  <h2 class="head2"> Voluntary Project Leaders </h2>
+  
+  <div class="people">
+    <div class="pic long">
+      <img src="{{  url_for('static', filename='img/LordSputnik.jpg') }}"/>
+    </div>
+    <div>
+      <h4><strong>BookBrainz</strong> - Ben Ockmore | LordSputnik</h4>
+      <p> Ben is one of the two project leads and the main programmer for <a href="https://bookbrainz.org">BookBrainz</a>, a volunteer project doing for books what MusicBrainz does for music. He started programming C++ 9 years ago, and now knows Python
+and Node.js too, although he is currently studying for a degree in Electronics Engineering at the<a href="http://www.surrey.ac.uk/"> University of Surrey </a>. Ben has been editing MusicBrainz for the past 4 years, attending the last three
+summits, and has contributed code to<a href="https://bitbucket.org/lazka/mutagen"> mutagen</a> and <a href="http://beets.io/"> beets</a> besides BookBrainz.
+<br/><br/>
+In his spare time, Ben likes to design quadrotors,<a href="http://www.surrey.ac.uk/mes/study/ug/projects/teamsurtes/about/index.htm"> racing cars</a>, artificial intelligence, <a href="http://sourceforge.net/projects/opengrow/">vegetation simulators</a> and music software. 
+However, none of them are finished yet!
+<br/><br/>
+Check out his GitHub profile: <a href="https://github.com/LordSputnik">https://github.com/LordSputnik </a></p>
+    </div>
+  </div>
+  
+  <div class="people">
+    <div class="pic reo">
+      <img src="{{  url_for('static', filename='img/Leftmost.jpg') }}"/>
+    </div>
+    <div>
+      <h4><strong>BookBrainz</strong> - Sean Burke | Leftmost</h4>
+      <p> Sean is one of the two project leaders for <a href="https://bookbrainz.org">BookBrainz</a>. He speaks Irish and is currently trying to learn to play the bodhrán.</p><br/><br/><br/>
```

Comment:
by hand. The bio was so short that it kind of swallowed alastairp's and it looked really bad and needed extra space.